presto-docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -0,0 +1,80 @@
+=============================
+Secure Internal Communication
+=============================
+
+The Presto cluster can be configured to use secured communication. Communication
+between Presto nodes can be secured with encryption, authentication and authorization.
+
+.. Note: At this step only encryption is supported
+
+Encrypted Internal Communication Configuration
+----------------------------------------------
+
+Encryption is configured in `config.properties` file.
+Encryption properties are identical for all of the presto node roles (coordinator, worker)
+and must be set on every node in the cluster.
+
+To enable encryption for the Presto internal communication follow the next steps:
+
+1. Disable HTTP endpoint.
+
+.. code-block:: none
+
+    http-server.http.enabled=false
+
+.. warning::
+
+    Leaving HTTP endpoint enabled leads to a security hole.
+
+2. Specify node's domains. Every node must have it's own domain configured.
```

Comment:
Do you mean domain or FQDN here? Domain implies a container for hosts, whereas coordinator.example.com is a host, not a container.